api/filter/accesslog/accesslog.proto

```diff
@@ -50,26 +50,52 @@ message AccessLogCommon {
   // downstream byte received (i.e. time it takes to receive a request).
```

Comment:
Can you qualify above that `start_time` is when the first downstream byte is received? Since most of the durations are relative to this, it could be good to be explicit.